src/components/calendar-date/calendar-date.twig

```diff
@@ -1,7 +1,7 @@
 {% set datetime = datetime|default('now') %}
 {% set days = days|default(1) %}
 
-<time class="c-calendar-date c-calendar-date--{{datetime|date('F')|lower}}"
+<time class="c-calendar-date c-calendar-date--{{datetime|date('F')|lower}}{% if class %} {{class}}{% endif %}"
```

Comment:
Am I missing it, or is this update used in the demo?

Reply:
It was at one point, I changed some things, and decided it was still a useful addition. Should have called it out in my PR description, though.